server/repository/migrations/sqlite/2022-02-23T10-00_create_number_trigger/up.sql

```diff
@@ -1,20 +0,0 @@
-CREATE TRIGGER number_insert_trigger
```

Comment:
@andreievg is this the correct way to go about it? Might be that we do want numbers in the change log (some future process might want to look for them... doubt it though!) - perhaps the synchronise process should just filter the table out?